### PR TITLE
✨[Feature Request]: Automatic assign

### LINKED ISSUE #1070: ✨[Feature Request]: Automatic assign
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

 if the project owner or mentor , comments 
/a , it will assign to the particular issue created contributor.

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Anjaliavv51**
#1070

## Description

if the project owner or mentor , comments
/a , it will assign to the particular issue created contributor.

## Related Issues

- Closes #1070

## Type of PR

- [X] (documentation)

## Checklist

- [X] I have gone through the [contributing guide](https://github.com/Anishkagupta04/RAPIDOC-HEALTHCARE-WEBSITE-/)
- [X] I have updated my branch and synced it with project `main` branch before making this PR
- [X] I have performed a self-review of my code
- [X] I have tested the changes thoroughly before submitting this pull request.
- [X] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [X] I have commented my code, particularly in hard-to-understand areas.

### CODE REVIEW

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR.

**Comment by Anjaliavv51** [comment]
@varshith257 please review the PR

File: .github/workflows/assignlabel.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+name: Assign Issue to Creator
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  assign-issue:
+    runs-on: ubuntu-latest
+    if: contains(github.event.comment.body, '/a')
```
Comment (varshith257) [inline .github/workflows/assignlabel.yml:10]:
```suggestion
    if: contains(github.event.comment.body, '/assign')
```

**Review by varshith257 — CHANGES_REQUESTED** [review]

File: .github/workflows/assignlabel.yml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            const { data: repoPermission } = await github.rest.repos.getCollaboratorPermissionLevel({
+              owner,
+              repo,
+              username: commenter,
+            });
+            
+            const allowedPermissions = ['admin', 'maintain', 'write'];
+            const hasPermission = allowedPermissions.includes(repoPermission.permission);
+            
+            console.log(`Commenter ${commenter} has permission: ${hasPermission}`);
```
Comment (varshith257) [inline .github/workflows/assignlabel.yml:30]:
```suggestion
```

**Review by varshith257 — CHANGES_REQUESTED** [review]

**Pull request merged**